package.json

```diff
@@ -17,13 +17,15 @@
   "dependencies": {
     "app-module-path": "^2.2.0",
     "axios": "^0.21.1",
+    "bootstrap": "^5.1.3",
```

Comment:
Why Bootstrap? Have you experimented with any other CSS frameworks?

Reply:
Yeah.. I've learnt a bit of Bootstrap and the video tutorial I was watching was using it, so I thought of installing it in case I would use it.
I did start coding with it but then I didn't use it in the end. 
I've only have experience with CSS and Bootstrap.